crates/bevy_render/src/camera/projection.rs

```diff
@@ -213,6 +233,12 @@ pub struct OrthographicProjection {
     ///
     /// Defaults to `1000.0`
     pub far: f32,
+    // If true, the far plane will be used for culling. If false, the far plane will be ignored.
+    ///
+    /// If true, objects farther from the camera than the far value will not be visible.
+    ///
+    /// Defaults to a value of `false`.
+    pub use_far_culling: bool,
```

Comment:
I'd rather `far` be made `Option<f32>` to better model the far plane than use a separate boolean.

Reply:
I agree.

Reply:
Is the Option really better? This will still default to some value (currently `0`) when computing projections and frustums. This results in a "far plane behind near plane" view frustum with a tiny volume. This isn't how people normally visualize or think about projections / view frustums. Projections are constructed with a near and a far value. View frustums are always "watertight" with 6 sides. From my perspective having a reasonable default value in front of the near value, with a separate knob to to control culling makes a lot more sense. Although culling doesn't feel like a projection configuration. Seems like configuration on a subset of things that have a projection (ex: camera or light config, or some shared "culled" concept).

@superdump, @james7132, @IceSentry, @NotoriousENG

Reply:
It also seems reasonable that future scenarios will want to take the far plane into account, with or without far plane culling disabled. Tying them together seems like a lossy abstraction.